packages/libs/react-ui/src/components/Select/Select.tsx

```diff
@@ -0,0 +1,65 @@
+import { SystemIcon } from '../Icons';
+
+import {
+  containerClass,
+  containerClassDisabled,
+  iconClass,
+  selectClass,
+  selectContainerClass,
+} from './Select.css';
+
+import classNames from 'classnames';
+import React, { FC, forwardRef } from 'react';
+
+export interface ISelectProps
+  extends Omit<
+    React.HTMLAttributes<HTMLSelectElement>,
+    'as' | 'disabled' | 'children' | 'className' | 'onChange' | 'value'
+  > {
+  children: React.ReactNode;
+  icon?: (typeof SystemIcon)[keyof typeof SystemIcon];
+  disabled?: boolean;
+  value: string[] | string | number;
+  onChange: React.ChangeEventHandler<HTMLSelectElement>;
+  ref?: React.ForwardedRef<HTMLSelectElement>;
+}
+
+export const Select: FC<ISelectProps> = forwardRef<
+  HTMLSelectElement,
+  ISelectProps
+>(function Select(
+  {
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    icon: Icon,
+    disabled = false,
+    children,
+    ...rest
+  },
+  ref,
+) {
+  return (
+    <div
+      className={classNames(
+        containerClass,
+        disabled ? containerClassDisabled : '',
+      )}
+      data-testid="kda-select"
+    >
+      <div className={selectContainerClass}>
+        {Icon && (
+          <span className={iconClass}>
+            <Icon size="md" />
+          </span>
+        )}
+        <select
+          ref={ref}
+          className={selectClass}
+          disabled={Boolean(disabled)}
```

Comment:
Does this really need to be cast to a boolean? The prop type should already enforce this, right?

Reply:
disabled is string in HTML so when we extend from HTML select element it expects to pass string